### PR TITLE
Add fast-react-static-renderer to example operations repos

### PR DESCRIPTION
**Pull request by bitPablo**
# Description

Adds the fast-react-static-renderer example operations repository to the documentation. 

Fixes #161

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation update

# How Has This Been Tested?

N/A

**Logs**

N/A

**Test Configuration**:
* BitOps image tag used:
* Hardware:
* Tool:

# Checklist:
This is existing code that I reviewed, but I did not modify the content.

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: docs/examples/bitops+fast+react+static+renderer/global-tools/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# global-tools
```
Comment (mickmcgrath13) [inline docs/examples/bitops+fast+react+static+renderer/global-tools/README.md:1]: Remove this file. I think it was copy/paste from a previous project

File: docs/examples/bitops+fast+react+static+renderer/dev/terraform/cloudfront.tf [hunk 2]
```diff
@@ -0,0 +1,87 @@
+# Cloudfront S3 for redirect to www.
```
Comment (mickmcgrath13) [inline docs/examples/bitops+fast+react+static+renderer/dev/terraform/cloudfront.tf:1]: delete this file. it was part of early testing.

**Review by mickmcgrath13 — CHANGES_REQUESTED** [review]
a few minor changes. otherwise, it looks good!

**Comment by bitPablo** [comment]
Performed requested changes.

**Review by mickmcgrath13 — APPROVED** [review]

**Pull request merged**